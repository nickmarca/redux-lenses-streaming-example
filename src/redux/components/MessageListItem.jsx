import React from 'react';

function MessageListItem({ label, value }) {
    return (
        <div className="column is-2">
            <div>{label}</div>
            {value}
        </div>
    );
}

export default MessageListItem;
