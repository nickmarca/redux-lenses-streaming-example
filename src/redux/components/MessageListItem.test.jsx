import React from 'react';
import { create } from 'react-test-renderer';
import MessageListItem from './MessageListItem';

describe('MessageListItem', () => {
    it('matches the snapshot', () => {
        const messageListItem = create(<MessageListItem />);

        expect(messageListItem.toJSON()).toMatchSnapshot();
    });
});
