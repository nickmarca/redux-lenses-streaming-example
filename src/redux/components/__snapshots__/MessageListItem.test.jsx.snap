// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageListItem matches the snapshot 1`] = `
<div
  className="column is-2"
>
  <div />
</div>
`;
